server/server.js

```diff
@@ -27,3 +34,8 @@ app.listen(port, (error) => {
         console.info(`==> 🌎  Listening on port ${port}. Open up http://localhost:${port}/ in your browser.`);
     }
 });
+
+app.post('/api/newProblem', (req, res) => {
+    console.log('posted: ', req.body);
```

Comment:
for Node.js debugging please use https://www.npmjs.com/package/debug instead of console.log()